.travis.yml

```diff
@@ -23,14 +30,17 @@ install:
   - popd
 
 before_script:
+  - sudo sed -i -e '/local.*peer/s/postgres/all/' -e 's/peer\|md5/trust/g' /etc/postgresql/11/main/pg_hba.conf
+  - sudo systemctl restart postgresql@11-main
   - createdb terasoluna-gfw-functionaltest --locale=C --encoding=UTF8 --template=template0
   - mvn -B dependency:copy -Dartifact=org.codehaus.cargo:cargo-daemon-webapp:${CARGO_DAEMON_WEBAPP_VERSION}:war -DoutputDirectory=./target/.
   - java -jar ./target/cargo-daemon-webapp-${CARGO_DAEMON_WEBAPP_VERSION}.war &
 
 script:
   - mvn -B xml:check-format
   - mvn -B formatter:validate
-  - mvn -B -U sql:execute -pl terasoluna-gfw-functionaltest-initdb
+  - mvn -B -U sql:execute -pl terasoluna-gfw-functionaltest-initdb -Ddb.url=jdbc:postgresql://127.0.0.1:5433/terasoluna-gfw-functionaltest
+  - sed -i -e "s/5432/5433/" *env/configs/travis/*/context.xml
```

Comment:
- [ ] Why not use `PGPORT` ?

Reply:
Omission of correction. use `PGPORT`.  
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/895/commits/9c849911f177e44f063998fb4ebcc5fd99a47bdd